### PR TITLE
Project 1

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Project - 1: initialize
Features introduced are as follows:
1. React, Redux structure
2. React components
3. React router

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Categories from './components/Categories';
+
+const App = () => (
+  <div>
+    <h1>Welcome to BookStore</h1>
+    <Router>
+      <Link to="/books">Books</Link>
+      <Link to="/categories">Categories</Link>
+      <Switch>
+        <Route path="/books">
```
Comment (Davidosky007) [inline src/App.js:16]: * Kindly create a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
image

File: src/components/Books.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => {
+	const books = [
```
Comment (Davidosky007) [inline src/components/Books.jsx]: * I suggest you create a book component to be imported into Books.jsx for rendering each book which would also have a dummy button (remove) which isn't expected to work for now [Use the project big picture as guide ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @SamanAtashi 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* Kindly check bellow for comments and try to fix them 🙏 Note that styling isn't required just rendering of the necessary components 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**